packages/combo-box/src/vaadin-combo-box-dropdown.js

```diff
@@ -209,22 +209,6 @@ export class ComboBoxDropdown extends PolymerElement {
     }
   }
 
-  /**
-   * Gets the index of the item with the provided label.
-   * @return {number}
-   */
-  indexOfLabel(label) {
```

Comment:
**note:** https://github.com/vaadin/web-components/pull/3944 should be updated after this change.